core/sql/sqlcomp/CmpSeabaseDDLcommon.cpp

```diff
@@ -891,6 +901,10 @@ short CmpSeabaseDDL::createMDdescs(MDDescsInfo *&trafMDDescsInfo)
   return 0;
 
  label_error:
+
+  // When debugging, you can look at the breadCrumb variable to figure out
+  // why you got here.
+
```

Comment:
Could you somehow include the breadcrumb variable in an error message, or log the information to the masterexe log in the "logs" directory?